### PR TITLE
Resolves issues #176 and #329

### PR DESCRIPTION
**Pull request by 7yl4r**
I'm new to all of these backend technologies, but these frontend fixes are a nice simple start for me.

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -55,4 +55,9 @@ $(document).ready(function() {
     var $li = $(this).parent("li").toggleClass('open');
     return false;
     });
+
+    // re-show custom text option that may have been previously hidden
+    $('.close').click(function(){
```
Comment (tsujigiri) [inline app/assets/javascripts/application.js]: Should the selector be more specific? It's a fix for one specific place, right?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def previous_or_next_page(page, text, classname)
   end
 
   def page_navigation_links(pages)
-    will_paginate(pages, class: 'pagination', inner_window: 2, outer_window: 0, renderer: BootstrapLinkRenderer, previous_label: '&larr;'.html_safe, next_label: '&rarr;'.html_safe)
+    will_paginate(pages, class: 'pagination', inner_window: 2, outer_window: 0, renderer: BootstrapLinkRenderer, previous_label: '&larr;'.html_safe, next_label: '&rarr;'.html_safe, page_links: false)
```
Comment (tsujigiri) [inline app/helpers/application_helper.rb]: It's better to put unrelated changes into individual pull requests. Makes it easier when looking through the history to figure out what was done why.

Comment (7yl4r) [inline app/helpers/application_helper.rb]: Thanks for both suggestions. I'll open a new PR for will_paginate and will a second one with a more specific selector string for the other issue.
